### ISSUE TITLE
Fitune SegUnet

### ISSUE THREAD

**Issue by jcwang123** [krishnabits001/domain_specific_cl#1]
There is some question about fintuning the segmentation model on "mmwhs":
1. The training loss is hard to converge. We set "--no_of_tr_imgs tr8 --comb_tr_imgs c1"  but the training loss only decrease to 0.5x, and the val_score is 0 in all epoches.
2. We found the augmentation will change the original value in label array, thus we use a function to re-map the value:
def ReMod(x, dataset = 'mmwhs'):
    '''
    for tf-onehot
    '''
    if dataset == 'mmwhs':
        mapp_v = np.array([  0, 205, 420, 500, 550, 600, 820, 850])
    x = np.repeat(np.expand_dims(x, axis=-1), len(mapp_v), -1)
    x = np.argmin(np.abs(x - mapp_v),axis=-1)
    return x

**Comment by jcwang123** [comment]
I've solved this problem by modifying the label of valid set.

**Comment by ElegantLin** [comment]
@jcwang123 Hi, Can you reproduce the result, and do you know how to pre-process the mmwhs dataset? I met some problems when I am trying to reproduce the result. The number I got is not as high as that given in the paper. Thanks for your help.

**Comment by jcwang123** [comment]
> @jcwang123 Hi, Can you reproduce the result, and do you know how to pre-process the mmwhs dataset? I met some problems when I am trying to reproduce the result. The number I got is not as high as that given in the paper. Thanks for your help.

Same as you. It's hard to reproduce the equal result presented in the paper.

**Comment by krishnabits001** [comment]
Hi Zonglin and @jcwang123 ,

I have upload the pre-processed and cropped MMWHS datasets in the below link. Please try with these volumes to run the experiments.
https://polybox.ethz.ch/index.php/s/len0MjehIMw8Kdu


Regards,
Krishna Chaitanya

**Comment by ElegantLin** [comment]
Hi Krishna,

Thanks for your reply. I would have a try :-)

Zonglin
